dev/com.ibm.ws.jdbc_fat_v41/test-applications/basicfat/src/jdbc/fat/v41/web/BasicTestServlet.java

```diff
@@ -915,6 +926,123 @@ public void getSingleConnectionAfterAbort() throws Exception {
+        assertEquals(2, getPoolSize("jdbc/ds2abort"));
+        try {
+            int i = 0;
+            int size = getPoolSize("jdbc/ds2abort");
+            while (size > 1) {
+                ++i;
+                Thread.sleep(1000);
+                size = getPoolSize("jdbc/ds2abort");
+                if (i > 120) {
+                    break;
+                }
+            }
+        } finally {
+            c2.close();
+            c1.close();
+            assertEquals(1, getPoolSize("jdbc/ds2abort"));
+        }
+    }
+
+    /**
+     * Function to just get and close a connection
+     *
+     * This function should be called right after testAbortedConnectionDestroyedAuto to
+     * ensure that the aborted connection was removed from the pool and existing
+     * connection is still usable.
+     */
+    public void getSingleConnectionAfterAbortAuto() throws Exception {
+        assertEquals(1, getPoolSize("jdbc/ds2abort"));
+        Connection c = ds2abort.getConnection();
+        c.close();
+        assertEquals(1, getPoolSize("jdbc/ds2abort"));
+    }
+
+    /**
+     * Test using maxInUseTime="1s"
+     */
+    @Test
+    @AllowedFFDC({ "java.security.PrivilegedActionException",
+                   "javax.resource.ResourceException" })
+    public void getSingleConnectionAfterAbortAuto1Second() throws Exception {
+        InitialContext ctx = new javax.naming.InitialContext();
+        DataSource ds2abort1s = null;
+        try {
+            ds2abort1s = (DataSource) ctx.lookup("jdbc/ds2abort1s");
+        } catch (NamingException n) {
+            // expected exception from lookup
+        }
+        assertNull(ds2abort1s);
+    }
+
+    /**
+     * Test using maxInUseTime="0"
+     *
+     * @throws Exception
+     */
+    @Test
+    @AllowedFFDC({ "java.security.PrivilegedActionException",
+                   "javax.resource.ResourceException" })
+    public void getSingleConnectionAfterAbortAuto0Seconds() throws Exception {
+        InitialContext ctx = new javax.naming.InitialContext();
+        DataSource ds2abort0 = null;
+        try {
+            ds2abort0 = (DataSource) ctx.lookup("jdbc/ds2abort0");
+            Connection c = ds2abort0.getConnection();
+            c.close();
+        } catch (NamingException n) {
+            // expected exception from lookup
+        }
+        assertNull(ds2abort0);
+    }
+
+    /**
+     * Test using maxInUseTime="-1"
+     *
+     * @throws Exception
+     */
+    @Test
+    public void getSingleConnectionAfterAbortAutoMinus1() throws Exception {
+        Connection c = ds2abortMinus1.getConnection();
+        c.close();
+    }
+
+    /**
+     * Test using maxInUseTime="60s"
+     *
+     * @throws Exception
+     */
+    @Test
+    public void getSingleConnectionAfterAbortAuto60Seconds() throws Exception {
+        Connection c = ds2abort60s.getConnection();
+        c.close();
+    }
+
+    /**
+     * Test using maxInUseTime="1m"
```

Comment:
There is no point in testing both
maxInUseTime="60s"
and
maxInUseTime="1m"
especially when the test takes so long to run.
The duplication would only be testing the ibm:type=duration({UNIT}) capability of Liberty config.

Even with just 1 test, I'd recommend designating it for FULL mode.